instrumentation/thrift/thrift-common/library/src/main/java/io/opentelemetry/instrumentation/thrift/common/ThriftRpcAttributesGetter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.thrift.common;
+
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcAttributesGetter;
+import javax.annotation.Nullable;
+
+public enum ThriftRpcAttributesGetter implements RpcAttributesGetter<ThriftRequest> {
+  INSTANCE;
+
+  @Override
+  public String getSystem(ThriftRequest request) {
+    return "thrift";
```

Comment:
You should also create a pr to add `thrift` to `rpc.system` values in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-spans.md. See https://github.com/open-telemetry/semantic-conventions/commit/7c1aa90f279a9901f9542f6562df0217611d9ccd for inspiration. It is possible that the value should be `apache_thrift` instead of `thrift` but I'll leave this for the semantic conventions folks to decide.